src/api/index.js

```diff
@@ -1,4 +1,5 @@
 import { statusMapper } from '../constants';
+import axios from 'axios';
```

Comment:
It should be better to use `instance` from frontend-components package, so we have all the interceptors.
```suggestion
import { instance } from '@redhat-cloud-services/frontend-components-utilities/interceptors/interceptors';
```